Formula/sleuthkit.rb

```diff
@@ -11,37 +12,25 @@ class Sleuthkit < Formula
     sha256 "994034b47b0992fa5c651787de90a548edf5a2ccddee7eb4e7a96bd74b23b470" => :el_capitan
   end
 
-  option "with-jni", "Build Sleuthkit with JNI bindings"
-  option "with-debug", "Build debug version"
-
-  depends_on "afflib" => :optional
-  depends_on "libewf" => :optional
-
-  if build.with? "jni"
-    depends_on :java
-    depends_on "ant" => :build
-  end
+  depends_on "ant" => :build
+  depends_on "afflib"
+  depends_on :java
```

Comment:
Any idea if this is compatible with Java 10+?

Reply:
I've been looking in the upstream docs to see if they do or don't but I haven't found anything yet. Doesn't seem to be any issues about it in the repo either.

Reply:
I assume CI has the latest Java installed, so presumably/hopefully it would blow if there were any issues, but I'll build it locally to check.

Reply:
This warning:
```
retrieve-deps:
[ivy:resolve] :: Apache Ivy 2.3.0-rc2 - 20121105223351 :: http://ant.apache.org/ivy/ ::
[ivy:resolve] :: loading settings :: file = /private/tmp/sleuthkit-20180525-96814-39n1b8/sleuthkit-4.6.1/bindings/java/ivysettings.xml
[ivy:resolve] :: resolving dependencies :: org.sleuthkit#datamodel;working@test.local
[ivy:resolve] 	confs: [default]
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.ivy.util.url.IvyAuthenticator (file:/usr/local/var/homebrewcache/java_cache/.ant/lib/ivy.jar) to field java.net.Authenticator.theAuthenticator
WARNING: Please consider reporting this to the maintainers of org.apache.ivy.util.url.IvyAuthenticator
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

Plus a bunch of deprecations like this:
```
compile:
    [javac] Compiling 66 source files to /private/tmp/sleuthkit-20180525-96814-39n1b8/sleuthkit-4.6.1/bindings/java/build
    [javac] /private/tmp/sleuthkit-20180525-96814-39n1b8/sleuthkit-4.6.1/bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java:7069: warning: [deprecation] finalize() in Object has been deprecated
    [javac] 	protected void finalize() throws Throwable {
    [javac] 	               ^
```

But it does build, at least.

Reply:
Thanks for checking that @DomT4!